docs/how-to-guides/block-tutorial/extending-the-query-loop-block.md

```diff
@@ -0,0 +1,195 @@
+	attributes: {
+		/** ...variation attributes */
+		namespace: 'my-plugin/books-list',
+	},
+	isActive: [ 'postType', 'namespace' ],
+}
+```
+
+Like so, Gutenberg will know that it is your specific variation only in the case it matches your custom namespace! So convenient!
+
+## Extending the query
+
+Even with all of this, your custom post type might have unique requirements: it might support certain custom attributes that you might want to filter and query for, or some other query parameters might be irrelevant or even completely unsupported! We have build the Query Loop block with such use-cases in mind, so let's see how you can solve this problem.
+
+### Disabling irrelevant or unsupported query controls
+
+Let's say you don't use at all the `sticky` attribute in your books, so that would be totally irrelevant to the customization of your block. In order to not confuse the users as to what a setting might do, and only exposing a clear UX to them, we want this control to be unavailable. Furthermore, let's say that you don't use the `author` field at all, which generally indicates the person who has added that post to the database, instead you use a custom `bookAuthor` field. As such, not only keeping the `author` filter would be confusing, it would outright break your query.
+
+For this reason, the Query Loop block supports a property called `allowedControls` which accepts an array of keys of the controls we want to whitelist. By default, we accept all the controls, but as soon as we provide an array to this property, we want to be specific and whitelist only the controls which are going to be relevant for us!
+
+As of version 13.9, the following controls are available:
+
+-   `inherit` - Shows the toggle switch for allowing the query to be inherited directly from the template.
+-   `postType` - Shows a dropdown of available post types.
+-   `order` - Shows a dropdown to select the order of the query.
+-   `sticky` - Shows a checkbox to only display sticky posts.
+-   `taxQuery` - Shows available taxonomies filters for the currently selected post type.
+-   `author` - Shows an input field to filter the query by author.
+-   `search` - Shows an input filed to filter the query by keywords.
+
+In our case, the property would look like this:
+
+```js
+{
+	/** ...variation properties */
+	allowedControls: [ 'inherit', 'order', 'taxQuery', 'search' ],
+}
+```
+
+Notice that we have also disabled the `postType` control: when the user selects our variation, why show them a confusing dropdown to change the post type? On top of that it might break the block as we can implement custom controls, as we'll see shortly.
+
+### Adding additional controls
+
+Because our plugin uses custom attributes that we need to query, we want the users of our block to be able to select those instead of the ones we have just disabled from the core inspector controls. We can do this via a [React HOC](https://reactjs.org/docs/higher-order-components.html), like so:
```

Comment:
Maybe some small rewarding of `be able to select` to something we need to add our own controls to handle our custom attributes and a link to here: https://developer.wordpress.org/block-editor/reference-guides/filters/block-filters/? The example below is still valuable.